lib/event_store_client/broker.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+module EventStoreClient
+  class Broker
+    def call(subscriptions)
+      subscriptions.each do |subscription|
+        new_events = connection.consume_feed(subscription.stream, subscription.name)
```

Comment:
What about latency if we will have many subscriptions - I'm not sure, I can explain my worries on call

Reply:
Ok, let's call tomorrow about it.